src/importexport/capella/tests/capella_tests.cpp

```diff
@@ -111,7 +111,7 @@ TEST_F(Capella_Tests, capTest8) {
 }
 TEST_F(Capella_Tests, capTestTuplet2) {
     capReadTest("testTuplet2");
-}                                                                           // generates different beaming with respect to the original
+}                                                               // generates different beaming with respect to the original
```

Comment:
When at it anyway... I'm not a big fan of these huge amounts of whitespace. I'd prefer it like this:
```suggestion
TEST_F(Capella_Tests, capTestTuplet2) {
    // generates different beaming with respect to the original
    capReadTest("testTuplet2");
}
```

Reply:
OK